scheduler/test/cook/test/mesos/util.clj

```diff
@@ -393,21 +393,50 @@
             _ (Thread/sleep 5)
             end-time (Date.)
             states [(name state)]
-            name (constantly true)]
+            namet (constantly true)
+            namef (constantly false)
+            ]
```

Comment:
The close bracket should be on the previous line

Reply:
@scrosby Thoughts on this?

Reply:
@scrosby Thoughts on this?

Reply:
Fixed in recent push.